src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+
+namespace Iot.Device.ServoMotor
+{
+    /// <summary>
+    /// Represents a rotary actuator or linear actuator that allows for precise control of angular or linear position.
+    /// </summary>
+    public class ServoMotor : IDisposable
+    {
+        private PwmChannel _pwmChannel;
+        private readonly int _maximumAngle;
+        private readonly int _maximumPulseWidthMicroseconds;
+        private readonly int _minimumPulseWidthMicroseconds;
+        private readonly double _angleInMicroseconds;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
+        private ServoMotor(int maximumAngle, int minimumPulseWidthMicroseconds, int maximumPulseWidthMicroseconds)
```

Comment:
the more I look at it the more I think we should add new unit for angles with internal representation being radians. The only trick part is angles are in modulo space and -180..180 are relatively commonly used so we should probably create an issue on how to design it